src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FC, useState, useEffect, useRef, RefObject } from 'react';
+
+import Button from 'components/Button';
```

Comment:
Use Material UI's [`Button`](https://material-ui.com/components/buttons/#buttons-with-icons-and-label) component here.